SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -728,9 +728,15 @@ FNetworkGUID FSpatialNetGUIDCache::GetNetGUIDFromUnrealObjectRefInternal(const F
 		FNetworkGUID OuterGUID;
 
 		// Recursively resolve the outers for this object in order to ensure that the package can be loaded
-		if (ObjectRef.Outer.IsSet())
+		if (ObjectRef.Outer.IsSet() && ObjectRef.Outer.GetValue() != FUnrealObjectRef::NULL_OBJECT_REF)
 		{
 			OuterGUID = GetNetGUIDFromUnrealObjectRef(ObjectRef.Outer.GetValue());
+
+			if (!OuterGUID.IsValid())
+			{
+				// Couldn't resolve the outer, most likely because it's a dynamic actor that we haven't received yet.
```

Comment:
Should this be an error?

Reply:
No, returning an empty GUID here will be treated as a regular unresolved reference.